src/nn/Model.js

```diff
@@ -155,6 +163,11 @@ export default class Model {
     if (!this._validateOperandList(outputs)) {
       throw new Error(`Invalid outputs ${outputs}`);
     }
+    if (type === OperationCode.ADD) {
+      if (this._isOperandQuantized(this._operands[inputs[0]])　||　this._isOperandQuantized(this._operands[inputs[1]])) {
+        this._isQuantized = true;
+      }
+    }
```

Comment:
Why not checking when adding operand?

Reply:
The same as above, I put the code here since I only check the operands for `add` operation.